docs/example_workflows/basic_workflow.yml

```diff
@@ -0,0 +1,20 @@
+name: My workflow
+on:
+  push:
+  pull_request:
+jobs:
+  check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Setup Starknet Foundry
+        uses: foundry-rs/setup-snfoundry@v3
+
+      - name: Setup Scarb
+        uses: software-mansion/setup-scarb@v1
+        with:
+          scarb-lock: ./hello_starknet/Scarb.lock
```

Comment:
Not related to this PR, but should we add this to our workflows? It won’t really speed things up, but we may get rid of the errors with restoring cache that occur in every run.